apps/meeting/server.js

```diff
@@ -15,18 +15,34 @@ let protocol = 'http';
 let options = {};
 
 const chime = new AWS.Chime({ region: 'us-east-1' });
-const alternateEndpoint = process.env.ENDPOINT;
-if (alternateEndpoint) {
-  console.log('Using endpoint: ' + alternateEndpoint);
-  chime.createMeeting({ ClientRequestToken: uuid() }, () => {});
-  AWS.NodeHttpClient.sslAgent.options.rejectUnauthorized = false;
-  chime.endpoint = new AWS.Endpoint(alternateEndpoint);
+// Create ans AWS SDK Chime object. Region 'us-east-1' is globally available.
+const chimeRegional = new AWS.ChimeSDKMeetings({ region: 'us-east-1' });
+const chimeRegionalEndpoint = process.env.REGIONAL_ENDPOINT || 'https://meetings-chime.us-east-1.amazonaws.com';
+
+if (chimeRegionalEndpoint) {
+  console.log('Using regional endpoint', chimeRegionalEndpoint);
+  chimeRegional.endpoint = new AWS.Endpoint(chimeRegionalEndpoint);
 } else {
+  console.log('Using global endpoint ');
   chime.endpoint = new AWS.Endpoint(
     'https://service.chime.aws.amazon.com/console'
   );
 }
 
+// return regional API just for Echo Reduction for now.
+function getClientForMeeting(meeting) {
+  if (
+    meeting &&
```

Comment:
You can avoid this by using optional chaining feature in JS:
```
meeting &&
    meeting.Meeting &&
    meeting.Meeting.MeetingFeatures &&
    meeting.Meeting.MeetingFeatures.Audio &&
    meeting.Meeting.MeetingFeatures.Audio.EchoReduction === 'AVAILABLE'
```
to
```
meeting?.Meeting?.MeetingFeatures?.Audio?.EchoReduction === 'AVAILABLE'
```

Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
fixed

Reply:
i will have to put it back because `?.` generates the following error
```
SyntaxError: Unexpected token '.'
     at wrapSafe (internal/modules/cjs/loader.js:1054:16)
     at Module._compile (internal/modules/cjs/loader.js:1102:27)
     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
     at Module.load (internal/modules/cjs/loader.js:986:32)
     at Function.Module._load (internal/modules/cjs/loader.js:879:14)
     at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
     at internal/main/run_main_module.js:17:47
```